spec/controllers/api/v1/tokens_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::TokensController, type: :controller do
+  let(:user) { create(:user) }
```

Comment:
let's try to leave out lets. Try to change it to something like
`def user
    @user ||= ....
  end`